src/pe_source/pe_scripts.py

```diff
@@ -0,0 +1,129 @@
+"""A tool for gathering pe source data.
+
+Usage:
+    pe-source DATA_SOURCE [--log-level=LEVEL] [--orgs=ORG_LIST] [--cybersix-methods=METHODS] [--soc_med_included]
```

Comment:
Underscores are not used for any of the other options:
```suggestion
    pe-source DATA_SOURCE [--log-level=LEVEL] [--orgs=ORG_LIST] [--cybersix-methods=METHODS] [--soc-med-included]
```

Reply:
Fixed here: [d8e46a6](https://github.com/cisagov/pe-source/pull/1/commits/d8e46a6c95a0d185990945b7ba9d347bdeb1524c) and [3a2e262](https://github.com/cisagov/pe-source/pull/1/commits/3a2e2620cbe58191f1652fca4b3adcfe78db6ba8)